packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -0,0 +1,297 @@
+// Based on:
+// https://github.com/microsoft/fast/blob/6bce27d0b2d654650b8751bf055f5e3b5a4f9250/packages/web-components/fast-foundation/src/tabs/tabs.ts
+// The code is heavily modified such that a diff against the original is useless.
+// Primary differences are:
+// - Reimplemented active tab indicator and removed attribute to disable it
+// - Separated tab focus and tab selection, i.e. can focus a different tab than the selected one
+// - Removed everything related to tab panels
+// - Removed support for vertical tab orientation
+// - Removed change event
+// - Conforms to our linter rules
+import { attr, observable } from '@microsoft/fast-element';
+import {
+    keyArrowLeft,
+    keyArrowRight,
+    keyEnd,
+    keyEnter,
+    keyHome,
+    keySpace,
+    uniqueId
+} from '@microsoft/fast-web-utilities';
+import {
+    DesignSystem,
+    StartEnd,
+    applyMixins,
+    StartEndOptions,
+    FoundationElementDefinition,
+    FoundationElement
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import { template } from './template';
+import type { AnchorTab } from '../anchor-tab';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-tabs': AnchorTabs;
+    }
+}
+
+export type TabsOptions = FoundationElementDefinition & StartEndOptions;
+
+/**
+ * A nimble-styled set of anchor tabs
+ */
+export class AnchorTabs extends FoundationElement {
+    /**
+     * The id of the active tab
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: activeid
+     */
+    @attr
+    public activeid = '';
+
+    /**
+     * @internal
+     */
+    @observable
+    public tabs!: HTMLElement[];
+
+    /**
+     * A reference to the active tab
+     * @public
+     */
+    public activetab: HTMLElement | undefined;
+
+    /**
+     * A reference to the tablist div
+     * @internal
+     */
+    public tablist!: HTMLElement;
+
+    private tabIds: string[] = [];
+    private readonly tabAttributeMutationObserver = new MutationObserver(
+        mutations => {
+            mutations.forEach(mutation => {
+                if (
+                    mutation.type === 'attributes'
+                    && mutation.attributeName === 'aria-disabled'
+                ) {
+                    this.setTabs();
+                }
+            });
+        }
+    );
```

Comment:
Discussed and can remove the mutation observer, the goal of it was to prevent rendering when a tabs is configured to mark a disabled control as active. Instead will render the disabled + active state (alternatively if we consider that an unsupported state then it should be documented and render "sanely" without additional complexity introduced for the case).

Reply:
Removed the mutation observer and updated the logic to allow a disabled tab to be the active tab. I had to add a line of CSS to gray out the active indicator in that case. The `nimble-tabs` story doesn't render the selection indicator when you set the first tab to both active and disabled, though. I think it actually would show it if you just set the active tab disabled without refreshing the whole page, but that's just what Storybook does when you change a setting.